symbolic-examples/compile_to_wasm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cabal clean
+
+# Compile to wasm
+cabal --with-compiler=wasm32-wasi-ghc-9.12 --with-hc-pkg=wasm32-wasi-ghc-pkg-9.12 --with-hsc2hs=wasm32-wasi-hsc2hs-9.12 build
```

Comment:
Instead of specifying `buildable: False` for executables in `.cabal`, I think you can build just a library here, like this:
```bash
cabal --with-compiler=... build symbolic-examples
```